src/lib.rs

```diff
@@ -608,12 +639,25 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
     /// }
     /// ```
     pub fn extend_from_vob(&mut self, other: &Vob<T>) {
+        self.copy_into_with_offset(other, 0);
+    }
+
+    /// Copy blocks from other into self, respecting offset.
+    ///
+    /// block_offset * bits_per_block should be less than other.len
+    #[inline]
+    fn copy_into_with_offset(&mut self, other: &Vob<T>, block_offset: usize) {
+        debug_assert!(
```

Comment:
I think we can just simplify this to `debug_assert!(block_offset * bits_per_block::<T>() <= other.len())` (not sure we need the message).